runtime/runtime/src/state_viewer/mod.rs

```diff
@@ -124,6 +124,25 @@ impl TrieViewer {
                 requested_account_id: account_id.clone(),
             });
         }
+        match get_account(state_update, account_id)? {
+            Some(account) => {
+                let code_len = get_code(state_update, account_id, Some(account.code_hash()))?
+                    .map(|c| c.code.len() as u64)
+                    .unwrap_or_default();
+                if account.storage_usage() > Account::MAX_ACCOUNT_DELETION_STORAGE_USAGE + code_len
+                {
+                    return Err(errors::ViewStateError::AccountStateTooLarge {
+                        requested_account_id: account_id.clone(),
+                    });
+                }
```

Comment:
How then I should clean up my state, if I can't view it either?

This will likely break the berryclub wayback machine that relies on the prefix query.

We should bump this limit to something higher since this is an RPC call